tools/verdict/com.ge.research.osate.verdict/plugin.xml

```diff
@@ -99,7 +117,21 @@
 	            style="push"
 	            tooltip="Click to change CRV Settings">
 	       </command>
-		</menu>                                   	                         
+		</menu>      
+		<menu id="GsnMenu" label="&amp;Assurance Cases (GSN)" >
+		   <command
+                commandId="com.ge.research.osate.verdict.commands.runGsn"
+                label="&amp;Create GSN"
+                style="push"
+                tooltip="Create GSN Fragment for selected requirement">
+          </command> 
+          <command
+                commandId="com.ge.research.osate.verdict.commands.gsnSettings"
+                label="&amp;GSN Settings"
```

Comment:
Change the "GSN Settings" -> "Assurance Case Settings"

Reply:
This change is for future SACM settings so that GSN and SACM can be on the same setting panel.

Reply:
Done